hub/etc/httpd/conf.d/max-requests.conf

```diff
@@ -1,4 +1,3 @@
 # Keep number of workers/threads low to avoid exhausting postgres connections
 # This should work for both prefork (RHEL 7) and event (Fedora) MPMs
 MaxRequestWorkers 40
-ThreadsPerChild 10
```

Comment:
This could be protected by some sort of IfModule - the internet has both
`<IfModule event.c>` and `<IfModule mpm_event_module>` - not sure which one is right or if both work. If you just remove it you'll get a single httpd worker process. Which is probably just fine for osbs-box, though going to be more prone to contention on the python GIL or any other process-global resources.

Reply:
From the apache docs: 
> The module argument can be either the module identifier or the file name of the module, at the time it was compiled. For example, rewrite_module is the identifier and mod_rewrite.c is the file name. If a module consists of several source files, use the name of the file containing the string STANDARD20_MODULE_STUFF.

Reply:
This seems to work (or at least not cause httpd to crash):
```
<IfModule mpm_event_module>
ThreadsPerChild 10
</IfModule>
```